src/Event/Listeners/OracleSessionInit.php

```diff
@@ -53,7 +53,11 @@ public function postConnect(ConnectionEventArgs $args) : void
             if ($option === 'CURRENT_SCHEMA') {
                 $vars[] = $option . ' = ' . $value;
             } else {
-                $vars[] = $option . " = '" . $value . "'";
+                if (is_numeric($value)) {
```

Comment:
Would it make sense to check if the value `is_int()`? Why would someone need to pass `10000` as `"10000"`?

Reply:
I explain my case in comment you prefert the function is_int() ?

```
 if ($option === 'CURRENT_SCHEMA' || is_int($value)) {
     $vars[] = $option . ' = ' . $value;
 } else {
     $vars[] = $option . " = '" . $value . "'";
 }
```

Reply:
The case makes sense. The above does as well.